serializers/src/glTF/2.0/babylon.glTFExporter.ts

```diff
@@ -161,7 +191,7 @@ module BABYLON.GLTF2 {
                     let scale = 1;
 
                     for (let j = 0; j < stride; ++j) {
-                        if (j === (stride - 1) && !useRightHandedSystem) {
+                        if (j === (stride - 1) && this.convertToRightHandedSystem) {

```

Comment:
This seems error-prone. Is it possible to calculate the min max after converting to right handed?

Reply:
Yeah, I can do that

Reply:
I now calculate the right-handed position vector before getting the min and max value from it.